.github/workflows/swift_package_test.yml

```diff
@@ -84,7 +84,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        swift_version: ['5.9', '6.0', 'nightly', 'nightly-6.0']
+        swift_version: ['5.9', '5.10', '6.0', 'nightly']
```

Comment:
Is there a reason we are dropping `nightly-6.0`?

Reply:
No, just forgot to add it back in after I removed it so testing uses fewer of my free GitHub Action minutes.